.eslintrc

```diff
@@ -0,0 +1,80 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "mocha": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "rules": {
+    "curly": "error",
+    "eqeqeq": "error",
+    "guard-for-in": "error",
+    "no-bitwise": "error",
+    "no-buffer-constructor": "error",
+    "no-caller": "error",
+    "no-console": "off",
+    "no-extra-boolean-cast": "off",
+    "no-undef": "error",
+    "no-unused-vars": [
+      "error",
+      {
+        "args": "none"
+      }
+    ],
+    "no-useless-escape": "off",
+    "semi": [
+      "error",
+      "always"
+    ],
+    "quotes": [
+      "error",
+      "single",
+      {
+        "allowTemplateLiterals": true
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": [
+        "**/tests/**",
+        "**/test/**"
+      ],
+      "globals": {
+        "_": true,
+        "after": true,
+        "afterAll": true,
+        "afterEach": true,
+        "assert": true,
+        "before": true,
+        "beforeAll": true,
+        "beforeEach": true,
+        "browser": true,
+        "by": true,
```

Comment:
What is `by` for?

Reply:
`by` belongs to `selenium-webdriver` 
It is for testing `elements` when doing `e2e` test mimicking user actions